src/goto-symex/goto_symex.h

```diff
@@ -524,7 +524,7 @@ class goto_symext
   {
     target.validate(ns, vm);
   }
-  validation_modet global_validation_mode;
+  bool run_validation_checks;
```

Comment:
With the exception that I'd place this textually next to the other Boolean variables that store options, yes, this is how I thought it could be done.